supplementary_style_guide/style_guidelines/specific-doc-types.adoc

```diff
@@ -136,20 +136,27 @@ If available, inform users of alternative capabilities and workarounds.
 
 [discrete]
 ===== Deprecation notice
+[subs="+quotes"]
 ----
-In <product> <release>, <name of capability> is deprecated. Bug fixes and support are provided through the end of the <releasename or, if unknown, “a future”> life cycle. After which, no new feature enhancements are made. (Optional: You can use <alternative capability> instead.)
+In __<product_name> <release>__, __<name_of_capability_or_feature>__ is deprecated and is planned to be removed in the __<deprecation_timeline>__. Red Hat will provide bug fixes and support for this feature during the current release life cycle, but this feature will no longer receive enhancements and will be removed. As an alternative to __<name_of_capability_or_feature>__, you can use __<alternative_capability_or_feature_if_available>__ instead.
```

Comment:
"life cycle" should be "lifecycle" one word. I know most were here, but would you mind fixing that as well while you're in here?

Reply:
Thanks @bergerhoffer. I'll fix that.